docs/README.md

```diff
@@ -0,0 +1,155 @@
+# KFServing
+A KFService is a "unit of model serving". Users may have many KFServices, including different definitions across dev, test, and prod environments, as well as across different model use cases. KFServices are defined using the [Kubernetes Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) standard. 
+
+## Specifying a Service
+### Resource Definition
+This top level resource definition is shared by all Kubernetes Custom Resources.
+
+| Field       |  Value      | Description |
+| ----------- | ----------- | ----------- |
+| kind       | KFService                     | https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#types-kinds |
```

Comment:
Instead of using links directly here, how about use "[xxx](xxx)"? This may more clear.

Reply:
Good point. Let me resolve these.